search/filters/FulltextFilter.php

```diff
@@ -73,5 +73,18 @@ public function getDbName() {
 
 		return parent::getDbName();
 	}
+	
+	private function prepareColumns($columns) {
+		$class = '';
+		$parts = explode(',', $columns);
+		array_walk($parts, function(&$col, $key) use (&$class) {
```

Comment:
It might be clearer to just use `array_map` here instead of getting messy with references, which are rather error prone.